pkg/testutils/testing.go

```diff
@@ -13,7 +13,7 @@ import (
 func DSN() string {
 	dsn := os.Getenv("MYSQL_DSN")
 	if dsn == "" {
-		return "msandbox:msandbox@tcp(127.0.0.1:8030)/test"
+		return "msandbox:msandbox@tcp(127.0.0.1:3306)/test"
```

Comment:
This is unrelated, but I think we can just change this, it was tied to an old sandbox version.

Reply:
I do still use it for testing, but yes we can change it.